DynamicActiveQuery.php

```diff
@@ -216,7 +216,7 @@ public function createCommand($db = null)
             % (`?) \(! \s*?
                 ( [a-z_\x7f-\xff][a-z0-9_\x7f-\xff]* (?: \. [^.|\s]+)* )
                 (?:  \| (binary (?:\(\d+\))? | char (?:\(\d+\))? | time (?:\(\d+\))? | datetime (?:\(\d+\))? | date
-                        | decimal (?:\(\d\d?(?:,\d\d?)?\))?  | double (?:\(\d\d?,\d\d?\))?
+                        | decimal (?:\(\d\d?(?:,\d\d?)?\))?  | double (?:\(\d\d?,\d\d?\))? | numeric? | text ? |
                         | int(eger)? | (?:un)? signed (?:\s+int(eger)?)?)  )?
             \s*? !\) \1 %ix
 REGEXP;
```

Comment:
I was thinking to implement just 3 types for pg numeric, jsonb, and char/text which is the default. 
Postgres may have a hole lot of other types but in my mind that does not matter since we get json types back.
If anyone wants to typecast to anything else he can use (!dyncol|jsonb!)::postgres_data_type. 
I'm thinking of including that possibility to the documentation. In fact the type declaration with the format dyncol|type
was done for compatibility reasons.
You are right about removing it from maria's regexp. I will remove it asap and implement a regexp for pg.
I'll get back to you with bug report later today.